src/relations/database_provides.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Library containing the implementation of the database provides relation."""
+
+import json
+import logging
+
+from charms.data_platform_libs.v0.database_provides import (
+    DatabaseProvides,
+    DatabaseRequestedEvent,
+)
+from ops.framework import Object
+
+from constants import (
+    CREDENTIALS_SHARED,
+    DATABASE_PROVIDES_RELATION,
+    MYSQL_DATABASE_CREATED,
+    MYSQL_ROUTER_PROVIDES_DATA,
+    MYSQL_ROUTER_REQUIRES_APPLICATION_DATA,
+    PEER,
+    UNIT_BOOTSTRAPPED,
+)
+
+# from ops.model import Application
```

Comment:
Remove.

Reply:
removed in b22efe12e0d56b70d439b490de176040b0e1bd8f